login-workflow/src/new-architecture/screens/EulaScreen/EulaScreenBase.tsx

```diff
@@ -0,0 +1,73 @@
+/** eslint-ignore */
+import React, { useCallback, useState } from 'react';
+import { EulaScreenProps } from './types';
+import { WorkflowCard, WorkflowCardActions, WorkflowCardBody, WorkflowCardHeader } from '../../components/WorkflowCard';
```

Comment:
No need to import it from WorkflowCard, importing just from '../../components' will work, as we exported those from components.